xbmc/utils/Color.h

```diff
@@ -15,6 +15,15 @@ namespace UTILS
 
   typedef uint32_t Color;
 
+  struct ColorEx
```

Comment:
Any other name suggestion instead of colorex? It doesn't convey the struct intention well

Reply:
suggestions? perhaps `ColorInfo` is better?

Reply:
ColorInfo sounds good to me